index.html

```diff
@@ -329,17 +329,17 @@ <h2 class="connection m-0 p-l-15">
 
             </div>
         </section>
-        <section class="contact w-100 b-top-left-rounded">
+        <div class="contact w-100 b-top-left-rounded">
             <h2 class="w-100">Contact me</h2>
             <div class=" contact-text w-90 p-15 m-auto">
                 <p>
                     If you have an application you are interested in developing,
                  a feature that you need built or a project that needs coding. I’d love to help with it
                 </p>
                 <form class="contact-form w-100 m-auto">
-                    <input type="text" class="w-90" placeholder=" Your name" id="name"/>
-                    <input type="text" class="w-90" placeholder="natajayanj14@gmail.com" id="email"/>
```

Comment:
Great job so far, however, you should put a name attribute in your text field

Reply:
- [ ] It would make your comment more descriptive if you explain why the name attribute is important in the text field.

Reply:
- This issue is well addressed. Thank you for the changes you made. :+1: